packages/daimo-api/src/db/db.ts

```diff
@@ -179,6 +179,14 @@ export class DB {
 
           UNIQUE (address, push_key)
         );
+
+        --
+        -- Jumbo Indexer setup for tracking processed blocks
+        --
+        CREATE TABLE IF NOT EXISTS daimo_index (
+          chain_id BIGINT NOT NULL,
+          latest_block_num BIGINT NOT NULL, 
+        );
```

Comment:
```
CREATE TABLE IF NOT EXISTS index.daimo_index
```

Reply:
easy way add remaining tables. go to TablePlus, select all the IndexDB tables, Export > uncheck everything except Export Structure > can add here